docs/architecture/012-services.md

```diff
@@ -17,13 +18,20 @@ was reached.
 
 ## Decision
 
-Each service represents the public API for a specific part of REMS.
-This public API can be used to implement the HTTP API, but also from
-other parts of the system.
+Each service represents the public API for a specific part of REMS that is easy to call
+and handles dependencies to other items:
+- resource, workflow, ... structure
```

Comment:
This list could be a bit clearer. Are these examples of services? Perhaps mention the namespaces as well?

Reply:
better now, thanks